### ISSUE TITLE
[Feature] Customize date format for inline blame

### ISSUE THREAD

**Issue by Jasonca2** [zielu/GitToolBox#299]
Git lens in VS Code offers support for a custom date format to be used when viewing inline blames. Could support for this be added in GitToolBox?

**Comment by zielu** [comment]
@Jasonica2 something totally custom in addition to predefined ones ?

**Comment by Jasonca2** [comment]
> @Jasonca2 something totally custom in addition to predefined ones ? 

Yes that is correct @zielu , and what would really be slick is a preview window/tooltip to show what the formatted date would look like in the settings pane. I think it would make sense to conform to common date time format codes such as those from the C standard (i.e., H for 24 hour clock or I for 12 hour clock, etc)
